### PR TITLE
feat: libzkp `v0.8.1`

### PR DESCRIPTION
**Pull request by silathdiir**
### Purpose or design rationale of this PR

- Upgrade zkevm-circuits and scroll-prover.
- Read VK from assets dir (and check after proves for the first time in scroll-prover).

### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [ ] build: Changes that affect the build system or external dependencies (example scopes: yarn, eslint, typescript)
- [ ] ci: Changes to our CI configuration files and scripts (example scopes: vercel, github, cypress)
- [ ] docs: Documentation-only changes
- [x] feat: A new feature
- [ ] fix: A bug fix
- [ ] perf: A code change that improves performance
- [ ] refactor: A code change that doesn't fix a bug, or add a feature, or improves performance
- [ ] style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] test: Adding missing tests or correcting existing tests

### Deployment tag versioning

Has `tag` in `common/version.go` been updated?

- [ ] No, this PR doesn't involve a new deployment, git tag, docker image tag
- [x] Yes

### Breaking change label

Does this PR have the `breaking-change` label?

- [ ] No, this PR is not a breaking change
- [x] Yes

### CODE REVIEW

**Comment by 0xmountaintop** [comment]
is this PR ready? @silathdiir

**Comment by silathdiir** [comment]
> is this PR ready? @silathdiir

Sorry, no. We haven't finished the zkevm-circuits testool bugs.
But the code of VK return is ready-for-review.

File: common/libzkp/impl/Cargo.toml [hunk 1]
```diff
@@ -20,8 +20,7 @@ maingate = { git = "https://github.com/scroll-tech/halo2wrong", branch = "halo2-
 halo2curves = { git = "https://github.com/scroll-tech/halo2curves.git", branch = "0.3.1-derive-serde" }
 
 [dependencies]
-prover = { git = "https://github.com/scroll-tech/scroll-prover", tag = "v0.7.5" }
-types = { git = "https://github.com/scroll-tech/scroll-prover", tag = "v0.7.5" }
```
Comment (silathdiir) [inline common/libzkp/impl/Cargo.toml:24]: `types` package has been merged into `prover` in scroll-prover.

**Comment by silathdiir** [comment]
Chunk VK is available when init.
Screen Shot 2023-09-06 at 10 14 57 AM

Batch VK is available when init.
Screen Shot 2023-09-06 at 10 15 13 AM

Chunk VKs are same after proving.
Screen Shot 2023-09-06 at 10 15 42 AM

Batch VKs are same after proving.
Screen Shot 2023-09-06 at 10 15 29 AM


Chunk and batch proofs are verified.
Screen Shot 2023-09-06 at 10 15 53 AM

**Comment by silathdiir** [comment]
Chunk VK is available when init.

Screen Shot 2023-09-07 at 10 08 41 AM

Batch VK is available when init.

Screen Shot 2023-09-07 at 10 09 14 AM

Proofs could be verified.

Screen Shot 2023-09-07 at 10 09 41 AM

**Review by georgehao — APPROVED** [review]

**Review by HAOYUatHZ — APPROVED** [review]

**Pull request merged**